Dockerfile

```diff
@@ -3,8 +3,8 @@ FROM golang:1.19 AS builder
 WORKDIR /app
 COPY . /app
 
-RUN CGO_ENABLED=0 GOOS=linux go build -a -o helpwave .
-
+ARG VERSION=undefined
```

Comment:
is the arg empty or set to "undefined"? the latter will suppress the warning in case there is no version set